knowledge-base/progressbar-add-multiple-labels.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Add Multiple Labels in a ProgressBar in Blazor
+description: Learn how to display two or more labels in a ProgressBar for Blazor.
+type: how-to
+page_title: How to Add Two Labels on a ProgressBar for Blazor - Current Progress and Remaining Value
```

Comment:
TL;DR The title is too long, I suggest using the same one as in the `title` field.

The `page_title` length should be no more than 50 characters because the total [maximum length](https://docs.telerik.com/style-guide/metadata) is 70 characters and 24 of the 70 characters are already consumed by default during build because we automatically append the ` - Telerik UI for Blazor` suffix.